pallets/parachain-system/src/lib.rs

```diff
@@ -1497,6 +1497,33 @@ impl<T: Config> UpwardMessageSender for Pallet<T> {
 	}
 }
 
+/// Something that can check the inherents of a block.
+#[deprecated = "use `cumulus-pallet-aura-ext::FixedVelocityConsensusHook` instead"]
```

Comment:
```suggestion
#[deprecated = "use `cumulus-pallet-aura-ext::FixedVelocityConsensusHook` instead"]
```

A couple nits:
  1. `type ConsensusHook` in parachain-system currently lives under a feature flag - therefore we are deprecating and asking users to enable a feature flag simultaneously. I'd recommend just adding a comment here for now rather than a deprecation notice.
  2. The comment should be general and not point directly to `aura_ext`.

Reply:
```suggestion
#[cfg_attr(feature = "parameterized-consensus-hook", deprecated = "Perform these checks using a `ConsensusHook` instead"]
```

Something like this?